docs/next/modules/en/pages/tutorials/quickstart.adoc

```diff
@@ -44,13 +44,13 @@ If you're customizing the installation parameters, please make sure that you are
 | Name | Version 
 
 | Cluster API Operator
-| `v0.17.0`
+| `v0.18.0`
```

Comment:
note: this assumes the new CAPI Operator minor release (not available yet) will be used in Turtles

Reply:
This is bumped to v0.17.1 patch version of CAPI operator